installer/release.sh

```diff
@@ -132,7 +132,7 @@ fi
 
 printf "; List of files\n%s\n%s\n%s\n%s\n%s\n" \
 	"Source: \"{#SourcePath}\\package-versions.txt\"; DestDir: {app}\\etc; Flags: replacesameversion; AfterInstall: DeleteFromVirtualStore" \
-	"Source: \"{#SourcePath}\\usr\\share\\git\\ReleaseNotes.css\"; DestDir: {app}\\usr\\share\\git; Flags: replacesameversion; AfterInstall: DeleteFromVirtualStore" \
+	"Source: \"{#SourcePath}\\..\\usr\\share\\git\\ReleaseNotes.css\"; DestDir: {app}\\usr\\share\\git; Flags: replacesameversion; AfterInstall: DeleteFromVirtualStore" \
```

Comment:
IIRC, you already removed this when you merged @robmen's PR 113 (or 117?) a few weeks back. I suppose if I remove it, it would be a null-op. And yes, I agree, this should be done one way or another. :smile: